projects/storefrontlib/cms-structure/seo/media-preconnect.service.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { inject, Injectable } from '@angular/core';
+import { PageMetaLinkService } from './page-meta-link.service';
+import { WindowRef, FeatureToggles } from '@spartacus/core';
+import { MediaService } from '../../shared/components/media/media.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MediaPreconnectService {
+  protected pageMetaLinkService = inject(PageMetaLinkService);
+  protected mediaService = inject(MediaService);
+  private featureToggles = inject(FeatureToggles);
+  private windowRef = inject(WindowRef);
+
+  addPreconnectLink(): void {
+    if (!this.featureToggles.createMediaPreconnectLinkInSsr) {
+      return;
+    }
+
+    if (!this.windowRef.isBrowser()) {
+      return;
+    }
```

Comment:
[MEDIUM]
My bad, I wrote this condition in the ticket description. But I introduced a bug.
Currently the feature is disabled in SSR. 😅 
But we wanted it to be enabled in SSR:
```suggestion
    if (this.windowRef.isBrowser()) {
      return;
    }
```

Reply:
UPDATE:
I have second thoughts, that this feature might be helpful also in case of CSR only storefront (e.g. CSR fallback). It will help to preconnect to media, in parallel to making the first occ call.

So:
- let's remove the check
- let's update the name of the feature toggle
- let's update JSdocs if needed (if they mentioned that the feature is meant only for SSR)